### PR TITLE
Let's add Blog Bear to the pack!

### PR DESCRIPTION
**Pull request by jennwrites**

### CODE REVIEW

File: _posts/2016-04-12-introducing-blog-bear.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+---
+
+By now you’re probably familiar with our animal friends the [Low-Profile Dog](http://hood.ie/animals/#low-profile-dog) (or LPD, as some of us like to call him) and the [Docs Chicken](http://hood.ie/animals/#docs-chicken). These animal friends play a huge part in making the community fun and welcoming and we are so happy to have collaborated with the very talented [Carolina Buzio](https://twitter.com/carolinabuzio) on their creation.
+
+Please join us in welcoming our newest animal mascot to the Hoodieverse, captured perfectly by editorial contributor [Annika Howard](https://github.com/NeekyRabbit).
+
+# Meet our latest animal friend: Blog Bear
+
+<a href="http://hoodiehq.github.io/hoodie-css/src/content_img/animals/blog-bear-1.png></a>
```
Comment (gr2m) [inline _posts/2016-04-12-introducing-blog-bear.md]: `href="...` is missing the closing `"`

Comment (gr2m) [inline _posts/2016-04-12-introducing-blog-bear.md]: also ... should there be the image in here?

**Comment by janl** [comment]
lgtm

File: _posts/2016-04-12-introducing-blog-bear.md (hunk 1 above)
Comment (gr2m) [inline _posts/2016-04-12-introducing-blog-bear.md]: I think what we want is `Blog Bear`?

File: _posts/2016-04-12-introducing-blog-bear.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+In a shady pine forest, close to the shore, a bear ambled slowly across a carpet of pine needles. 
+
+She stopped, sat back on her haunches, with a little picnic basket next to her. She opened the lid of the basket, took out a slice of strawberry cheesecake, and hooked out a green hoodie. Carefully she slipped her big paws into the sleeves and pulled a little red pencil from one pocket and a spiral-bound notebook from the other. 
+
+This was no ordinary bear. No, this was a bear with a lot to say and a lot to convey. The first Wednesday of every month you’ll find her hosting the BIT (Bears in Tech) breakfast with her buddies Baloo, Yogi, Paddington and Iorek Byrnison. She aspires to inspire and empower animals of all species: chickens, dogs, alpacas and more to contribute to making the web a better place. She’s got her ear to the ground and her paw on the pulse; always networking, making connections and gathering information. 
+
+Putting up the hood she turned to address us and in a gentle growl she asked, “Wwhat would the API of your dreams look like?” We knew we had found our Blog Bear.
```
Comment (gr2m) [inline _posts/2016-04-12-introducing-blog-bear.md]: Is `Wwhat` on purpose?

File: _posts/2016-04-12-introducing-blog-bear.md (hunk 1 above)
Comment (jennwrites) [inline _posts/2016-04-12-introducing-blog-bear.md]: You're right, I spaced and added the tag while I was writing the copy and never updated oops!

**Comment by janl** [comment]
screen shot 2016-04-12 at 17 10 13

File: _posts/2016-04-12-introducing-blog-bear.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+In a shady pine forest, close to the shore, a bear ambled slowly across a carpet of pine needles. 
+
+She stopped, sat back on her haunches, with a little picnic basket next to her. She opened the lid of the basket, took out a slice of strawberry cheesecake, and hooked out a green hoodie. Carefully she slipped her big paws into the sleeves and pulled a little red pencil from one pocket and a spiral-bound notebook from the other. 
+
+This was no ordinary bear. No, this was a bear with a lot to say and a lot to convey. The first Wednesday of every month you’ll find her hosting the BIT (Bears in Tech) breakfast with her buddies Baloo, Yogi, Paddington and Iorek Byrnison. She aspires to inspire and empower animals of all species: chickens, dogs, alpacas and more to contribute to making the web a better place. She’s got her ear to the ground and her paw on the pulse; always networking, making connections and gathering information. 
+
+Putting up the hood she turned to address us and in a gentle growl she asked, “Wwhat would the API of your dreams look like?” We knew we had found our Blog Bear. 
+
+# Welcome Blog Bear!
```
Comment (gr2m) [inline _posts/2016-04-12-introducing-blog-bear.md]: maybe link Blog Bear to `/animals/#blog-bear`?

File: _posts/2016-04-12-introducing-blog-bear.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+---
+
+By now you’re probably familiar with our animal friends the [Low-Profile Dog](http://hood.ie/animals/#low-profile-dog) (or LPD, as some of us like to call him) and the [Docs Chicken](http://hood.ie/animals/#docs-chicken). These animal friends play a huge part in making the community fun and welcoming and we are so happy to have collaborated with the very talented [Carolina Buzio](https://twitter.com/carolinabuzio) on their creation.
+
+Please join us in welcoming our newest animal mascot to the Hoodieverse, captured perfectly by editorial contributor [Annika Howard](https://github.com/NeekyRabbit).
+
+# Meet our latest animal friend: [Blog Bear](http://hood.ie/animals/#blog-bear)
+
+![Blog Bear](https://github.com/hoodiehq/hood.ie/blog/images/201604/blog-bear-1.png)
```
Comment (gr2m) [inline _posts/2016-04-12-introducing-blog-bear.md]: this image does not exist

**Pull request merged**